spa/src/utils/dynamicTableHelper.tsx

```diff
@@ -147,6 +159,28 @@ export const getGHSubscriptionsRows = (
 							</div>
 						</div>
 					),
+				},
+				{
+					key: cloudConnection.id,
+					content: (
+						<>
+							{/* TODO: Convert this into a dropdown */}
```

Comment:
Can you connect this to a ticket so that we don't forget it

Reply:
Added the ticket number.